components/properties/src/bc.rs

```diff
@@ -0,0 +1,71 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use crate::BidiDataSource;
```

Comment:
`question`: where is `BidiDataSource` defined? Why do you need a trait?

Reply:
I think it comes from the `unicode-bidi` crate. The import needs to be fixed.

Reply:
yes, it comes from `unicode-bidi` and I have fixed the import.

@zbraniecki , we need to implement to plugin the data from ICU4X to unicode-bidi.